.circleci/config.yml

```diff
@@ -18,6 +27,47 @@ jobs:
             tar -zxvf kubebuilder_${version}_linux_${arch}.tar.gz
             sudo mv kubebuilder_${version}_linux_${arch} /usr/local/kubebuilder
             export PATH=$PATH:/usr/local/kubebuilder/bin
-
       - run: make test
-      - run: make
+
+  run_e2e_tests:
+    machine:
+      image: ubuntu-1604:201903-01
+      docker_layer_caching: true
+
+    working_directory: ~/go/src/github.com/xridge/kubestone
+    steps:
+      - checkout
+      - run:
+          name: Upgrade golang to 1.12
+          command: |
+            sudo rm -rf /usr/local/go
+            VERSION=1.12.8
+            OS=linux
+            ARCH=amd64
+            curl -L -O https://dl.google.com/go/go${VERSION}.${OS}-${ARCH}.tar.gz
+            sudo tar -C /usr/local -xzf go$VERSION.$OS-$ARCH.tar.gz
+            go version
+      - run:
+          name: Install Kustomize
+          command: |
+            VERSION=3.1.0
+            OS=linux
+            ARCH=amd64
+            curl -L -O https://github.com/kubernetes-sigs/kustomize/releases/download/v${VERSION}/kustomize_${VERSION}_${OS}_${ARCH}
+            chmod +x kustomize_${VERSION}_${OS}_${ARCH}
+            sudo mv kustomize_${VERSION}_${OS}_${ARCH} /usr/local/bin/kustomize
+      - run:
+          name: Start e2e test
+          command: |
+            export GO111MODULE=on
+            export GOPATH=~/go
+            export PATH=$PATH:$GOPATH/bin
+            ./tests/e2e/e2e-test-in-kind.sh
```

Comment:
We should create a Makefile target for this command

Reply:
done